packages/codec/lib/abi-data/allocate/index.ts

```diff
@@ -7,7 +7,7 @@ import type * as Abi from "@truffle/abi-utils";
 
 import * as Import from "@truffle/codec/abi-data/import";
 import * as AbiDataUtils from "@truffle/codec/abi-data/utils";
-import Web3Utils from "web3-utils";
+import * as Web3Utils from "web3-utils";
```

Comment:
@jdevcs We should probably fix this on our end and give them option to import everything under `Web3Utils` for backward compatibility